var/spack/repos/builtin/packages/nvtop/package.py

```diff
@@ -7,16 +7,33 @@
 
 
 class Nvtop(CMakePackage, CudaPackage):
-    """Nvtop stands for NVidia TOP, a (h)top like task monitor for NVIDIA GPUs.
-    It can handle multiple GPUs and print information about them
-    in a htop familiar way."""
+    """Nvtop stands for Neat Videocard TOP, a (h)top like task monitor
+    for AMD and NVIDIA GPUS. It can handle multiple GPUs and print
+    information about them in a htop familiar way"""
 
     homepage = "https://github.com/Syllo/nvtop"
-    url      = "https://github.com/Syllo/nvtop/archive/1.1.0.tar.gz"
+    url      = "https://github.com/Syllo/nvtop/archive/refs/tags/2.0.1.zip"
 
-    version('1.1.0', sha256='00470cde8fc48d5a5ed7c96402607e474414d94b562b21189bdde1dbe6b1d1f3')
+    maintainers = ['marcost2']
+
+    version('2.0.1', sha256='ef18ce85d632eb1c22d3a3653976b2c088260039702df39fd0181f7cd3ae277d')
+    version('2.0.0', sha256='1651f34274c334a682f280dcb2f28d9642d44c7b22afe8c431cab91345b50f31')
+    version('1.2.2', sha256='543cbfdae3241fab1ea022402734c12e69d5988583193adaab69fdfae6e14c84')
+    version('1.2.1', sha256='197992cdd0e2e151fce91a7ba56f717e4d85b317c396001e8dbd84dc2ba363cd')
+
+    variant('support', values=('nvidia','amd'), default='nvidia,amd',multi=True,
+            description='Which GPU vendors to build support for')
 
     depends_on('ncurses')
+    depends_on('libdrm', when='support=amd')
 
     def cmake_args(self):
-        return [self.define('NVML_RETRIEVE_HEADER_ONLINE', True)]
+        args = []
+
+        if 'support=nvidia' in self.spec:
+            args.append('NVIDIA_SUPPORT=ON')
+
+        if 'support=amd' in self.spec:
+            args.append('AMDGPU_SUPPORT=ON')
+
+        return args
```

Comment:
I think you miss the `-D` to the CMake arguments. Also, you can write this as:
```suggestion
        return [
            self.define('NVIDIA_SUPPORT', self.spec.satisfies('support=nvidia')),
            self.define('AMDGPU_SUPPORT', self.spec.satisfies('support=amd'))
        ]
```
Beware of typos, since I didn't try the code above.